v1/api/swagger/components/schemas/Allocation.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
+description: allocation of servers or storage for each PCE service
+properties:
+  ResourceType:
+    type: string
+    description: Type of the resource the allocation inforamtion is listed. It is either Server or Volume
```

Comment:
Typo - "inforamtion". Should this read as "Type of the resource the allocation information is listed **for**."